drivers/watchdog/Kconfig.andes_atcwdt200

```diff
@@ -0,0 +1,15 @@
+# Kconfig Andes Watchdog configuration options
+#
+# Copyright (c) 2023 Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config WDT_ANDES_ATCWDT200
+	bool "Andes Watchdog driver"
+	default y
+	depends on DT_HAS_ANDESTECH_ATCWDT200_ENABLED
```

Comment:
Wouldn't it be simpler to have `default y if DT_HAS_ANDESTECH_ATCWDT200_ENABLED`?

Reply:
Thanks for your review, I understand what you mean.
But this style is reference to PR #48085 , so I am not sure which one is better

Reply:
I have the feeling (but have no numbers) that the style as it is right now is the default in the code base. I would say it's fine to keep it.